src/components/application_manager/test/commands/mobile/set_global_properties_test.cc

```diff
@@ -147,80 +193,69 @@ class SetGlobalPropertiesRequestTest
 
 TEST_F(SetGlobalPropertiesRequestTest,
        OnEvent_UIHmiSendSuccess_UNSUPPORTED_RESOURCE) {
-  MessageSharedPtr msg_vr = CreateFullParamsUISO();
-  (*msg_vr)[am::strings::msg_params][am::strings::vr_commands][0] =
-      "vr_command";
+  MessageSharedPtr msg_tts = CreateMsgWithCommonParams();
 
   utils::SharedPtr<SetGlobalPropertiesRequest> command =
-      CreateCommand<SetGlobalPropertiesRequest>(msg_vr);
+      CreateCommand<SetGlobalPropertiesRequest>(msg_tts);
 
   EXPECT_CALL(app_mngr_, RemoveAppFromTTSGlobalPropertiesList(kConnectionKey));
   const smart_objects::SmartObject* vr_help_title =
-      &((*msg_vr)[am::strings::msg_params][am::strings::vr_help_title]);
+      &((*msg_tts)[strings::msg_params][strings::vr_help_title]);
   const smart_objects::SmartObject* vr_help =
-      &((*msg_vr)[am::strings::msg_params][am::strings::vr_help]);
+      &((*msg_tts)[strings::msg_params][strings::vr_help]);
   const smart_objects::SmartObject* vr_help_prompt =
-      &((*msg_vr)[am::strings::msg_params][am::strings::help_prompt]);
+      &((*msg_tts)[strings::msg_params][strings::help_prompt]);
```

Comment:
@dcherniev, in my opinion, there is better to create a reference to `msg_params` like 
```C++
const smart_objects::SmartObject& msg_tts_params = (*msg_tts)[strings::msg_params];
// And then use it
const smart_objects::SmartObject* vr_help_title = 
    &(msg_tts_params[string::vr_help_title]);
const smart_objects::SmartObject* vr_help = 
    &(msg_tts_params[string::vr_help]);
const smart_objects::SmartObject* vr_help_prompt = 
    &(msg_tts_params[string::vr_help_prompt]);
```